.github/workflows/scheduled.yml

```diff
@@ -218,6 +218,40 @@ jobs:
       - name: Deploy to GitHub Pages
         id: deployment
         uses: actions/deploy-pages@v1
+  report-status:
+    name: Report deploy status
+    needs: [generate-cask, generate-core, generate-analytics, generate-samples, build, deploy]
+    if: ${{ always() && github.ref_name == 'master' }}
```

Comment:
```suggestion
    if: ${{ =github.ref_name == 'master' }}
```

Does this work?

Reply:
Without the `always()` function, this job won't run if previous dependent jobs do not have a `success` status.

Reply:
Cool good with me!